CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v1.12.1 (16/09/2021)
+
+**olp-sdk-dataservice-api**
+
+- Updated the `value` param in `UrlBuilder.appendQuery()` method. The value also can be an object like `{ [key: string]: string | number }`.
+- Updated APIs generated by the `Interactive V1`. Some interfaces have breaking changes  (https://github.com/heremaps/here-data-sdk-typescript/commit/70d7d8799de59c23157dc0437eef0f78a46c8b63).
```

Comment:
- Updated the `value` parameter in the `UrlBuilder.appendQuery()` method. You can also use this parameter as an object. For example, `{ [key: string]: string | number }`.
- Updated the APIs generated by Interactive API v1. Some interfaces have breaking changes: https://github.com/heremaps/here-data-sdk-typescript/commit/70d7d8799de59c23157dc0437eef0f78a46c8b63.